src/bci_build/templates.py

```diff
@@ -91,6 +91,8 @@
 <!-- {{ INFOHEADER }}-->
 <!-- OBS-AddTag: {% for tag in image.build_tags -%} {{ tag }} {% endfor -%}-->
 <!-- OBS-Imagerepo: obsrepositories:/ -->
+{%- if image.os_version.is_sle15 and image.name == "base" %}
+<!-- OBS-IgnorePackage: rpm -->{% endif %}
```

Comment:
Do we need this? I don't see it in [sles15-image.kiwi](https://build.suse.de/projects/SUSE:SLE-15-SP6:Update/packages/sles15-image/files/sles15-image.kiwi?expand=1) but there's this instead:
```xml
<!-- OBS-ExcludeArch: i586 -->
```

Reply:
Previous kiwi schemaversions allow `<package ... replaces="foobar"/>` but that's no longer part of the schema, so it has to be done a different way

Reply:
The `<!-- OBS-ExcludeArch: i586 -->` is only needed if this is part of the SLE project itself which provides the i586 arch as well.

Reply:
> Previous kiwi schemaversions allow `<package ... replaces="foobar"/>` but that's no longer part of the schema, so it has to be done a different way

I see, thanks!

> The `<!-- OBS-ExcludeArch: i586 -->` is only needed if this is part of the SLE project itself which provides the i586 arch as well.

Don't we want to publish the container from the SLE project though? Then we'd need this additional comment